packs/feats/halfling-luck.json

```diff
@@ -28,7 +28,33 @@
             "remaster": true,
             "title": "Pathfinder Player Core"
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "Note",
+                "outcome": [
+                    "failure",
+                    "criticalFailure"
+                ],
+                "predicate": [
+                    {
+                        "gt": [
+                            "halfling-luck",
+                            0
+                        ]
+                    }
+                ],
+                "selector": [
+                    "skill-check",
+                    "saving-throw"
+                ],
+                "text": "You can re-roll this check or saving throw.",
+                "title": "Halfling Luck"
+            },
+            {
+                "key": "RollOption",
+                "option": "halfling-luck:{item|frequency.value}"
+            }
+        ],
```

Comment:
Clever, but the incredible luck and guiding luck feats make this a much bigger pain in the ass

Reply:
I didn't even think about those future feats, looking at them I can see how they will make it much more difficult, hah.